pkg/bootloader/grub_test.go

```diff
@@ -111,15 +112,21 @@ var _ = Describe("Booloader", Label("bootloader", "grub"), func() {
 		grub = bootloader.NewGrub(cfg, bootloader.WithGrubDisableBootEntry(true))
 		Expect(grub.Install(rootDir, bootDir, "DEVICE_LABEL")).To(Succeed())
 
-		// Check everything is copied in boot directory
-		data, err := fs.ReadFile(fmt.Sprintf("%s/grub2/grub.cfg", bootDir))
+		// Check grub config in EFI directory
+		data, err := fs.ReadFile(filepath.Join(bootDir, "/EFI/BOOT/grub.cfg"))
 		Expect(err).To(BeNil())
 		Expect(data).To(Equal(grubCfg))
-		_, err = fs.Stat(fmt.Sprintf("%s/grub2/x86_64-efi/loopback.mod", bootDir))
+
+		data, err = fs.ReadFile(filepath.Join(bootDir, "/EFI/ELEMENTAL/grub.cfg"))
 		Expect(err).To(BeNil())
-		_, err = fs.Stat(fmt.Sprintf("%s/grub2/x86_64-efi/xzio.mod", bootDir))
+		Expect(data).To(Equal(grubCfg))
+
+		// Check everything is copied in boot directory
+		_, err = fs.Stat(filepath.Join(bootDir, "x86_64-efi/loopback.mod"))
```

Comment:
not sure about hardcoded architectures 🤔

Reply:
Hmm, I think it's fine since it's only for the tests. There was no platform set though, so the tests would fail when ran on another architecture. I added linux/amd64 as the default platform now and in the future we could consider running these tests for all supported platforms. (I see there is already a few tests for arm64 and risc-v as well).